pkg/controller/volume/persistentvolume/testing/testing.go

```diff
@@ -210,32 +211,32 @@ func (r *VolumeReactor) React(action core.Action) (handled bool, ret runtime.Obj
 		r.claims[claim.Name] = claim
 		r.changedObjects = append(r.changedObjects, claim)
 		r.changedSinceLastSync++
-		klog.V(4).Infof("saved updated claim %s", claim.Name)
+		logger.V(4).Info("Saved updated claim", "PVC", klog.KObj(claim))
 		return true, claim, nil
 
 	case action.Matches("get", "persistentvolumes"):
 		name := action.(core.GetAction).GetName()
 		volume, found := r.volumes[name]
 		if found {
-			klog.V(4).Infof("GetVolume: found %s", volume.Name)
+			logger.V(4).Info("GetVolume: found volume", "volumeName", volume.Name)
 			return true, volume.DeepCopy(), nil
 		}
-		klog.V(4).Infof("GetVolume: volume %s not found", name)
+		logger.V(4).Info("GetVolume: volume not found", "volumeName", name)
 		return true, nil, apierrors.NewNotFound(action.GetResource().GroupResource(), name)
 
 	case action.Matches("get", "persistentvolumeclaims"):
 		name := action.(core.GetAction).GetName()
 		claim, found := r.claims[name]
 		if found {
-			klog.V(4).Infof("GetClaim: found %s", claim.Name)
+			logger.V(4).Info("GetClaim: found claim", "PVC", klog.KObj(claim))
 			return true, claim.DeepCopy(), nil
 		}
-		klog.V(4).Infof("GetClaim: claim %s not found", name)
+		logger.V(4).Info("GetClaim: claim not found", "claimName", name)
```

Comment:
```suggestion
		logger.V(4).Info("GetClaim: claim not found", "PVC", klog.KRef(namespace (but which one?), name))
```

Disregard this comment if there is no namespace easily available here.

Reply:
done